integreat_cms/cms/templates/events/event_form_sidebar/venue_box.html

```diff
@@ -65,5 +66,13 @@
                 {% translate "Open on Google Maps" %}
             </a>
         </div>
+        <div id="poi-ajax-success-message"
+             class="w-full flex flex-wrap justify-center items-center bg-green-200 font-bold hidden">
```

Comment:
```suggestion
             class="bg-green-100 border-l-4 border-green-500 text-green-500 px-4 py-3 hidden">
```
I would style this the same way as our other success messages